node/tests/integration/main.rs

```diff
@@ -560,7 +560,7 @@ async fn assert_slot_status_raise(
     assert_eq!(
         game_account_storage.get_item(slot_index),
         RpoDigest::new([
-            Felt::from(slot_data.current_turn_index()),
+            Felt::from(slot_data.current_turn_index() + 13),

```

Comment:
Can we name this offset (13) ?

Reply:
Done in the latest PR
https://github.com/RizeLabs/aze-server/blob/c6a10c2ed0a19c3d41256e1f45b6d1c2c7338cfa/node/tests/integration/main.rs#L1014